### ISSUE TITLE
`cargo_lock::package::source::GitReference` no longer accessible in `8.0.0`

### ISSUE THREAD

**Issue by UebelAndre** [rustsec/rustsec#587]
Maybe I'm missing it but after https://github.com/rustsec/rustsec/pull/558 I don't see a re-export of [cargo_lock::package::source::GitReference](https://github.com/rustsec/rustsec/blob/cargo-lock/v8.0.1/cargo-lock/src/package/source.rs#L305-L316). If it's currently not accessible would it be possible to export it?

**Comment by tarcieri** [comment]
Yes, this is true, the `source` module should probably have remained `pub`, or it could be re-exported under `package`
